1_concepts/1_1_default_clone_copy/src/main.rs

```diff
@@ -1,3 +1,67 @@
+use std::fmt;
+
+#[derive(Copy, Clone, Default)]
+struct Point {
+    x: i32,
+    y: i32,
+}
+
+#[derive(Clone)]
+struct Polyline {
+    points: Option<Vec<Point>>,
```

Comment:
Why `Option`? Is empty `Vec` not enough to express `Point`s absence?

Reply:
Yes, empty `Vec` is enought